src/views/chats/Home/HomeChat.vue

```diff
@@ -131,8 +131,10 @@ export default {
     openModal(modalName) {
       this.$refs['home-chat-modals'].openModal(modalName);
     },
-    openModalFileUploader(files) {
-      this.$refs['home-chat-modals'].openModal('fileUploader', files);
+    openModalFileUploader(files, filesType) {
+      const teste = this.$refs['home-chat-modals'];
```

Comment:
I believe the variable name (`teste`) could be more descriptive, what do you think?

Reply:
Yep, that went unnoticed lol. I'll adjust!